src/watcher.js

```diff
@@ -0,0 +1,88 @@
+require('dotenv').config()
+const path = require('path')
+const Web3 = require('web3')
+const { connectWatcherToQueue } = require('./services/amqpClient')
+const { getBlockNumber } = require('./utils/utils')
+const processDeposits = require('./events/processDeposits')
+const processCollectedSignatures = require('./events/processCollectedSignatures')
+const processWithdraw = require('./events/processWithdraw')
+const { redis } = require('./services/redisClient')
+
+const config = require(path.join('../config/', process.argv[2]))
```

Comment:
Maybe we should add a check that `process.argv.length >= 3` here, with a proper error message.